src/data/boundary.json

```diff
@@ -1,4 +1,32 @@
 {
   "name": "Boundary",
-  "slug": "boundary"
+  "slug": "boundary",
+  "sidebar": {
```

Comment:
For many products, have also added the `sidebar` data necessary to get the landing pages rendering as expected. As with all content in this PR, this is intended as a first cut based on designs, and is not meant to reflect the final content state.